src/Scripting/Core/ScriptOptions.cs

```diff
@@ -19,11 +21,51 @@ public sealed class ScriptOptions
     {
         public static ScriptOptions Default { get; } = new ScriptOptions(
             filePath: "",
-            references: ImmutableArray<MetadataReference>.Empty,
+            references: GetDefaultMetadataReferences(),
             namespaces: ImmutableArray<string>.Empty,
             metadataResolver: RuntimeMetadataReferenceResolver.Default,
             sourceResolver: SourceFileResolver.Default);
 
+        private static ImmutableArray<MetadataReference> GetDefaultMetadataReferences()
+        {
+            string corlibDir = RuntimeMetadataReferenceResolver.DefaultCorLibDirectoryOpt;
+
+            if (corlibDir == null)
+            {
+                return ImmutableArray<MetadataReference>.Empty;
+            }
+
+            // Provide similar surface to mscorlib (netstandard 2.0).
+            // These references are resolved lazily. Keep in sync with list in core csi.rsp.
+            var files = new[]
```

Comment:
@terrajobst What do you think about this list? These options are the defaults for scripting scenarios https://github.com/dotnet/roslyn/wiki/Scripting-API-Samples. What I'm aiming for to give sensible default set of references for customers targeting netstandard 1.3+ (our scripting package targets 1.3). Something that roughly emulates netstandard 2.0 mscorlib.

Another question: as you can see we look for these assemblies in the directory that contains ```typeof(object).Assembly``` (see impl of ```RuntimeMetadataReferenceResolver.DefaultCorLibDirectoryOpt```). Although it is probably possible (is it?) to configure CoreCLR to load these assemblies from different directories I don't think that's common. If they indeed are the customer using scripting API in such CoreCLR host will need to specify custom ScriptOptions with proper references.